tests/test_virtual_chassis.py

```diff
@@ -224,7 +273,79 @@ def test_chassis_system_neigh(self, vct):
                 assert encap_index == sys_neigh_encap_index, "Encap index not sync-ed correctly"
                 
                 break
+            
+        # Verify programming of remote neighbor in asic db and programming of static route and static
+        # neigh in the kernel for the remote neighbor. The neighbor created in linecard 1  will be a 
```

Comment:
Would it be easy to also cover the case of neighbor deletion too?

Reply:
Yes. I'll add a test case for delete neighbor.

Reply:
Fixed. Added test case for system neigh delete